### PR TITLE
Use tor binaries from TorBrowser 8.0.3

### PR DESCRIPTION
**Pull request by freimair**
Updates netlayer dependency to version 0.4.7.2 which includes the tor-binaries extracted from the TorBrowser version 8.0.3 release a month or so ago.

Lets see if jitpack works...

### CODE REVIEW

**Comment by freimair** [comment]
jitpack does some weird things... onto it

File: build.gradle [hunk 1]
```diff
@@ -125,7 +125,7 @@ configure(project(':common')) {
 configure(project(':p2p')) {
     dependencies {
         compile project(':common')
-        compile('com.github.JesusMcCloud.netlayer:tor.native:0.4.7.2') {
+        compile('com.github.JesusMcCloud.netlayer:tor.native:0.4.7.1.1') {
```
Comment (ManfredKarrer) [inline build.gradle:128]: Why it was reverted to 4.7.1.1?

Comment (ManfredKarrer) [reply build.gradle:128]: Ah I saw https://github.com/JesusMcCloud/netlayer/releases/tag/0.4.7.1.1 was the latest release.

**Review by ManfredKarrer — APPROVED** [review]
utACK

File: build.gradle (hunk 1 above)
Comment (freimair) [reply build.gradle:128]: Yes, jitpack refused to build the correct commit  behind the 0.4.7.2 tag. It is a quick workaround until https://github.com/jitpack/jitpack.io/issues/3572 is resolved (if it ever gets resolved).

Furthermore, I do have a solution for #1761 almost ready. Hence, I am going to bump the netlayer-version to 0.6 anyways. And yes, the jitpack behaviour does bite us again - hence, 0.6. (All in agreement with Bernd)...

**Pull request merged**